tests/unit/command/test_dag.py

```diff
@@ -45,11 +45,11 @@ def graph(tmp_dir, dvc):
     return dvc.graph
 
 
-def test_build(graph):
+def test_build(graph):  # pylint: disable=W0621
```

Comment:
@JIoJIaJIu, you should not get this warning at all, as we use `pylint-pytest` plugin that understands fixture. Could you please check, if all of the dependencies for pylint are installed properly?